chef/data_bags/crowbar/migrate/neutron/022_seperate_ml2_drivers.rb

```diff
@@ -0,0 +1,57 @@
+def upgrade ta, td, a, d
+  if a['networking_mode']
+    a['ml2_type_drivers'] = a['networking_mode']
```

Comment:
> ml2_type_drivers is a list. So I guess this needs to be a['ml2_type_drivers'] = [a['networking_mode']]

Done in https://github.com/toabctl/barclamp-neutron/commit/e031434a8db081f91d103898fb8c5486eaf69621